tools/transformer/src/transform.js

```diff
@@ -17,6 +18,10 @@ let { doc, apiFileLocation } = getAPI(process.argv.slice(2));
 doc = applyDiscriminatorTransformations(doc);
 doc = applyOneOfTransformations(doc);
 doc = applyAllOfTransformations(doc);
-doc = applyModelNameTransformations(doc, "ApiAtlas", "View");
+
+doc = applyModelNameTransformations(doc, "ApiAtlas", "View", ignoredModelNames);
```

Comment:
I think the order of this operation matters, so if we have a model `ApiAtlasApi{name}View` this sequence will transform it into `{name}` which I don't know it's the behaviour we want.

We could first remove the prefix "Api", and then from the resulting spec remove the prefix "Atlas".

```suggestion
doc = applyModelNameTransformations(doc, "Api", "", ignoredModelNames);
doc = applyModelNameTransformations(doc, "Atlas", "", ignoredModelNames);
doc = applyModelNameTransformations(doc, "", "View", ignoredModelNames);
```

Reply:
It looks like the results output is the same. Will create follow-up PR.